test/config_test.py

```diff
@@ -0,0 +1,20 @@
+from laa_court_data_api_app.config.app import get_app_settings
+import os
+
+
+async def test_app_settings_returns_properties():
```

Comment:
Does this need to be an async method?

Reply:
No, it doesn't actually - the async is in there because of a previous misconception I had about using the BaseSettings class - will remove this. However, reckon I should keep the pytest.ini for future tests?